docs/content/help-support/issue-triage/avm-issue-triage.md

```diff
@@ -258,22 +258,36 @@ To look for Orphaned Modules:
 
+**Language agnostic steps**
+
 1. Submit a "deprecated module" issue by using the "[Deprecate AVM Module 🔴](https://aka.ms/avm/DeprecatedModule)" issue template.
 2. Make sure the &nbsp;<mark style="background-image:none;white-space: nowrap;background-color:#FBCA04;">Needs: Triage 🔍</mark>&nbsp; and the &nbsp;<mark style="background-image:none;white-space: nowrap;background-color:#000000;color:white;">Status: Module Deprecated 🔴</mark>&nbsp; labels are assigned to the issue and it is assigned to the "[AVM - Module Triage](https://github.com/orgs/Azure/projects/529)" GitHub project.
 3. Update the AVM Module Indexes, following the [process documented internally](https://dev.azure.com/CSUSolEng/Azure%20Verified%20Modules/_wiki/wikis/AVM%20Internal%20Wiki/684/Module-index-update-process).
-4. Place an information notice as per the below guidelines:
-    - In case of a Bicep module:
-      - Place the information notice - with the text below - in an `DEPRECATED.md` file, in the module's root.
-      - Run the [`utilities/tools/Set-AVMModule.ps1`](https://github.com/Azure/bicep-registry-modules/blob/main/utilities/tools/Set-AVMModule.ps1) utility with the module path as an input. This re-generates the module’s `README.md` file, so that the `README.md` file will also contain the same notice in its header.
-      - Make sure the content of the `DEPRECATED.md` file is displayed in the `README.md` in its header (right after the title).
-      - Publish a new patch version, having the updated `README.md` stating the module is deprecated.
-    - In case of a Terraform module:
-      - Place the information notice - with the text below - in the `README.md` file, in the module's root.
-      - Archive the module's repository on GitHub.
-    - Once the information notice is placed, submit a Pull Request.
-5. Keep the module's `-owners-` and `-contributors-` GitHub teams, as these will keep granting access to the source code of the module.
 
-Place the following information notice in the module's repository:
+**Bicep specific steps**
+
+4. Place an information notice as per the below guidelines:
+    1. Place the information notice - with the text below - in an `DEPRECATED.md` file, in the module's root.
+    1. Add a 'DEPRECATED - ' prefix to the `main.bicep`'s metadata description. For example, `metadata description = 'DEPRECATED - This module deploys a XYZ'` - i.e, add the 'DEPRECATED' prefix while keeping your original description. This description will be displayed to users of the VS-Code Bicep extension when searching for the module.
+    1. Run the [`utilities/tools/Set-AVMModule.ps1`](https://github.com/Azure/bicep-registry-modules/blob/main/utilities/tools/Set-AVMModule.ps1) utility with the module path as an input. This re-generates the module’s `README.md` file, so that the `README.md` file will also contain the same notice in its header.
```

Comment:
```suggestion
    1. Run the [`utilities/tools/Set-AVMModule.ps1`](https://github.com/Azure/bicep-registry-modules/blob/main/utilities/tools/Set-AVMModule.ps1) utility with the module path as an input. This re-generates the module’s `README.md` file, so that the `README.md` file will also contain the same notice in its header. For more instructions on how to use the script, please refer to the corresponding section in the[Contribution Guide](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/generate-bicep-module-files/)).
```